docs/tutorial/part-zero/index.md

```diff
@@ -92,11 +102,11 @@ npm install --global gatsby-cli
 
 If successfully installed, running `gatsby --version` should return a version number, and running `gatsby --help` will show different commands available to you using the `gatsby-cli` tool.
 
-## Create a site
+## Create a Gatsby site
 
-Now you can use the gatsby-cli tool to create your first Gatsby site. Using the tool, you can use “starters”, or partially built sites with some default configuration, to help you get moving faster on creating a certain type of site. The “Hello World” starter you’ll be using here is a starter with the bare essentials needed for a [Gatsby](/) site.
+Now you are ready to use the Gatsby CLI tool to create your first Gatsby site. Using the Gatsby CLI you can download “starters” (partially built sites with some default configuration already done) to help you get moving faster on creating a certain type of site. The “Hello World” starter you’ll be using here is a starter with the bare essentials needed for a Gatsby site.
```

Comment:
This is a bit wordy. Try "Now you are ready to use Gatsby CLI to create your first Gatsby site. Using the tool, you can download..."

And then remove the phrase "already done." Default configuration implies the same meaning.

Reply:
You are right I have made these changes in the new version